hw_17/mocha-test/calculator.spec.ts

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert';
+import { expect } from 'chai';
+import { Calculator } from '../src/calculator';
+
+let calculator: any;
+describe('Tests for calculator', () => {
+  before('Initialize class instance', () => {
+    calculator = new Calculator();
+  });
+
+  it('Should check the addition of numbers', () => {
+    assert.equal(calculator.addition(4, 2), 6, 'Addition is not as expected!');
```

Comment:
Why did you imported `expect` and the whole `chai` library itself if you don't use it anywhere in your code.